service/host.go

```diff
@@ -69,12 +69,22 @@ func (uis *UIServer) hostPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	var containers []host.Host
+	if h.HasContainers {
+		containers, err = h.GetContainers()
```

Comment:
I would expect that shadowing err here is fine, since it's never used outside of this scope. Is this really a lint error?